From 870974818ad55e03961fd15ee2ee9a445d2d90e9 Mon Sep 17 00:00:00 2001
From: Thomas Goirand <zigo@debian.org>
Date: Thu, 8 Oct 2015 11:06:37 -0700
Subject: Fixed Python 3 syntax error

 Note: no need to forward this upstream as the code in this file has gone from
 the master branch.
Forwarded: no
Last-Update: 2015-06-19

Patch-Name: fix-syntax-error-in-python3.patch
---
 amqp/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amqp/utils.py b/amqp/utils.py
index 900d2aa..bd74011 100644
--- a/amqp/utils.py
+++ b/amqp/utils.py
@@ -35,7 +35,7 @@ class promise(object):
     def __call__(self, *args, **kwargs):
         try:
             self.value = self.fun(
-                *self.args + args if self.args else args,
+                *(self.args + args if self.args else args),
                 **dict(self.kwargs, **kwargs) if self.kwargs else kwargs
             )
         except Exception as exc:
